### ISSUE TITLE
ReferenceError: primordials is not defined

### ISSUE THREAD

**Issue by snowman** [mayswind/AriaNg#479]
```bash
$ node --version
v13.5.0

$ npm --version
6.13.4
```

steps to reproduce:
```bash
$ git clone https://github.com/mayswind/AriaNg
$ cd AriaNg
$ npm install
$ npm install --global gulp-cli
$ gulp clean build-bundle
```

Error
```bash
fs.js:34
} = primordials;
    ^

ReferenceError: primordials is not defined
    at fs.js:34:5
    at req_ (/PathTo/AriaNg/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/PathTo/AriaNg/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/PathTo/AriaNg/node_modules/vinyl-fs/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:1139:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1159:10)
    at Module.load (internal/modules/cjs/loader.js:988:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1028:19)
    at require (internal/modules/cjs/helpers.js:72:18)
```

**Comment by mayswind** [comment]
try to use an older version of nodejs and npm?

**Comment by snowman** [comment]
solved by `$ nvm install v8.9.0`.

To list available Long-Term-Support nodejs version `$ nvm ls-remote | grep LTS`
